### PR TITLE
Some small refinements after some months of usage

### PR DESCRIPTION
**Pull request by Avaq**
# Motivation

Get rid of nuisances.

# Changes

See the commit messages. There's a motivation for each change in the commit message body.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ module.exports = {
     'func-style': [2],
     'class-methods-use-this': [2],
     'linebreak-style': [2],
-    'max-len': [2, {code: 100}],
+    'max-len': [1, {code: 100}],
```
Comment (dicearr) [inline index.js:65]: I do agree with the changes and rationale of this commit. Complying with the other rules, sometimes, can lead to non-trivial re-factors that we want, as developers, to address later on. However, I do think that `max-len` is quite easy to comply with, so it doesn't make sense to me to have it as a warning.

Comment (Avaq) [reply index.js:65]: Ah. You're right. I didn't mean to include line length; I mistook this this for the number of lines in a function.

Comment (Avaq) [reply index.js]: @dicearr updated

**Review by dicearr — APPROVED** [review]
I'm very happy with this changes, specially the last 3 commits. 🎉  🚀

**Review by philipmendels — APPROVED** [review]
Looks good to me. Side note: If the jsx preset is supposed to be used in React-TypeScript projects (I guess so), then I am not sure if some of the (new) rules make sense there. I know that `plugin:@typescript-eslint/recommended` already disables some rules from `eslint/recommended` but perhaps not from `eslint-plugin-react(/recommended)`. But I guess we can disable them later in the typescript preset when we encounter problems.

**Pull request merged**